### PR TITLE
Create cluster with specific name if not exists

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

File: mdb-realm/src/deployApps.ts [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
 
+        await deployCluster(config, appsPath);
+    } catch (error: any) {
+        core.setFailed(`An unexpected error occurred: ${error.message}\n${error.stack}`);
+    }
+}
+
+export async function deployCluster(config: EnvironmentConfig, appsPath: string): Promise<void> {
+    const clusterNames = config.useExistingCluster ? await getClusters(config) : [];
+    if (!clusterNames.includes(config.clusterName)) {
```
Comment (nirinchev) [inline mdb-realm/src/deployApps.ts:27]: Not sure I understand why that was moved to a different function.

File: update-changelog/.eslintrc.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
       "@typescript-eslint/func-call-spacing": ["error", "never"],
       "@typescript-eslint/no-array-constructor": "error",
       "@typescript-eslint/no-empty-interface": "error",
-      "@typescript-eslint/no-explicit-any": "error",
+      "@typescript-eslint/no-explicit-any": "off",
```
Comment (nirinchev) [inline update-changelog/.eslintrc.json:26]: Why was that changed?

**Review by nirinchev — APPROVED** [review]
Looks good - just small nits from me.

**Comment by nirinchev** [comment]
We should make the differentiator optional if cluster-name is provided.

File: mdb-realm/src/deployApps.ts (hunk 1 above)
Comment (desistefanova) [reply mdb-realm/src/deployApps.ts:27]: In order to be able to be called by the tests also.

File: update-changelog/.eslintrc.json (hunk 2 above)
Comment (desistefanova) [reply update-changelog/.eslintrc.json:26]: Because otherwise it doesn't accept having `catch (error: any)` where `error `is `any`.
